llama_stack/distribution/library_client.py

```diff
@@ -265,28 +280,30 @@ def _convert_body(self, path: str, body: Optional[dict] = None) -> dict:
         for param_name, param in sig.parameters.items():
             if param_name in body:
                 value = body.get(param_name)
-                converted_body[param_name] = self._convert_param(
+                converted_body[param_name] = self._convert_to_pydantic(
                     param.annotation, value
                 )
         return converted_body
 
-    def _convert_param(self, annotation: Any, value: Any) -> Any:
+    def _convert_to_pydantic(self, annotation: Any, value: Any) -> Any:
```

Comment:
is there a reason this has to be a class method? Cant it be a utility like convert_pydantic_to_json_value ?

Reply:
@raghotham totally it should be a free func. it was older code so I did not change it; will update